lib/src/backend.rs

```diff
@@ -73,15 +74,13 @@ pub struct Timestamp {
 
 impl Timestamp {
     pub fn now() -> Self {
-        Self::from_datetime(chrono::offset::Local::now())
+        Self::from_zoned(Zoned::now())
     }
 
-    pub fn from_datetime<Tz: chrono::TimeZone<Offset = chrono::offset::FixedOffset>>(
-        datetime: chrono::DateTime<Tz>,
-    ) -> Self {
+    pub fn from_zoned(zoned: Zoned) -> Self {
         Self {
-            timestamp: MillisSinceEpoch(datetime.timestamp_millis()),
-            tz_offset: datetime.offset().local_minus_utc() / 60,
+            timestamp: MillisSinceEpoch(zoned.timestamp().as_millisecond()),
+            tz_offset: zoned.time_zone().to_offset(zoned.timestamp()).seconds() / 60,
```

Comment:
A `Zoned` has an offset inside of it for its timestamp, so you shouldn't need to re-compute it here. [`Zoned::offset()`](https://docs.rs/jiff/latest/jiff/struct.Zoned.html#method.offset) should do the trick.